packages/framework/src/Component/Error/ErrorIdLogProcessor.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Shopsys\FrameworkBundle\Component\Error;
+
+use Monolog\LogRecord;
+use Symfony\Component\EventDispatcher\EventSubscriberInterface;
+use Symfony\Component\HttpKernel\KernelEvents;
+use Symfony\Contracts\Service\ResetInterface;
+
+final class ErrorIdLogProcessor implements EventSubscriberInterface, ResetInterface
+{
+    private ?string $errorId = null;
```

Comment:
What about using InMemoryCache instead of manually resetting the cache

Reply:
Like we said `Processor` should be as lightweight as possible, so I didn't change that